homeassistant/components/knx/binary_sensor.py

```diff
@@ -51,9 +51,14 @@ def is_on(self) -> bool:
     @property
     def extra_state_attributes(self) -> dict[str, Any] | None:
         """Return device specific state attributes."""
+        attr: dict[str, Any] = {}
+
         if self._device.counter is not None:
-            return {ATTR_COUNTER: self._device.counter}
-        return None
+            attr[ATTR_COUNTER] = self._device.counter
+        if self._device.last_telegram is not None:
+            attr[ATTR_SOURCE] = str(self._device.last_telegram.source_address)
+            attr[ATTR_LAST_KNX_UPDATE] = str(self._device.last_telegram.timestamp)
```

Comment:
What kind of time is this?

Reply:
In the lib it's a datetime object. Do you want me to convert it to a predefined format? If so, which format would this be?

Reply:
Times in state attributes must be UTC.